src/Form/RegisteredUserType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\RegisteredUser;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class RegisteredUserType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('pseudo')
+            ->add('firstname')
+            ->add('lastname')
+            ->add('phone')
+            ->add('address')
+            ->add('jobAddress')
+            ->add('ogr')
+            ->add('user')
+            ->add('subscription')
```

Comment:
i feel like you should have a form that creates the "User" entity associated with the "RegisteredUser" entity created here => "user" and "subscription" fields should not be here